dockerfiles/runtime_ubt18/Dockerfile

```diff
@@ -0,0 +1,75 @@
+# Download base image ubuntu 18.04
+FROM ubuntu:18.04
+
+ENV HTTPS_PROXY http://172.16.1.135:3128
```

Comment:
Not necessary?

Reply:
Fixed in [f973972](https://github.com/openxrlab/xrprimer/pull/39/commits/f9739722b97083b69836a4dc395c9cbabe4e62a6).